ui/src/CaseDetails.js

```diff
@@ -13,13 +20,17 @@ import PrintFulfilment from "./PrintFulfilment";
 import SensitiveData from "./SensitiveData";
 import { Link } from "react-router-dom";
 import SmsFulfilment from "./SmsFulfilment";
+import JSONPretty from "react-json-pretty";
```

Comment:
no such library in the `package.json`

Did you forget to specify `--save` when you ran `npm install react-json-pretty`?